addons/mail/models/mail_thread.py

```diff
@@ -4833,6 +4833,10 @@ def _get_mail_thread_data_attachments(self):
     def _get_allowed_message_post_params(self):
         return {"attachment_ids", "body", "email_add_signature", "message_type", "partner_ids", "subtype_xmlid"}
 
+    @api.model
+    def _get_allowed_message_update_params(self):
+        return {"attachment_ids", "body", "partner_ids", "scheduled_date"}
```

Comment:
Why are we adding `scheduled_date` here? It wasn't supported in the controller before this PR.

Reply:
It is used in `_message_update_content`, which means it can be posted through the controller, even if we don't do that for now. Since we're controlling the update params, I thought it would be cleaner to include it along with other params that could be passed to this method.

Reply:
I think it's in the private method only for internal usage, specifically NOT to be in the controller.

Reply:
Removed